### DISCUSSION TITLE
Project beta feedbacks

### DISCUSSION THREAD

**Discussion by Eywek** [community/community#6579] [Projects and Issues]
Hello,

I've started to use the new project beta and I have few things that could be improved imo:
- **Realtime:** When moving an item from another tab, we should see the update in realtime in the board (if this is already implemented it seems that it doesn't work for me)
- **Auto-import:** We should be able to automatically import to project, newly created issues in specific repositories
- **Link PR and issues:**: When we link a PR to issues, we could have only 1 card on the board view, like it's done on Zenhub
![Capture d’écran 2021-10-21 à 11 13 58](https://user-images.githubusercontent.com/6900054/138248105-9f152bb5-2769-4372-86bf-82c23e36f746.png)
- **Collapse in board view**: We should be able to collapse tabs in the board view
- **Hide a group:** We should be able to hide a group in the list view, only when all items in this group are in a specific status (i.e. hide my "Milestone 1" from the list view only when all items in this milestone are done)

**Comment by mikermcneil** [comment]
Related: https://github.com/github/feedback/discussions/10080
